Movie/Movie/CinemaDescriptionScreen/Presenter/CinemaDescriptionPresenter.swift

```diff
@@ -0,0 +1,11 @@
+// CinemaDescriptionPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Презентер экрана с подробностями о фильме.
+final class CinemaDescriptionPresenter: CinemaDescriptionPresenterProtocol {
+    // MARK: Public properties
+
+    var viewController: CinemaDescriptionViewProtocol!
```

Comment:
```suggestion
   weak var viewController: CinemaDescriptionViewProtocol!
```
поможет избежать ретейн цикла